CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## dev
 
+**Breaking changes**
+
+- Removed `pygdbmi.IoManager.make_non_blocking` from the public API; it's unrelated and was not meant to be public
```

Comment:
This is my interpretation but I'm not sure if it's correct (see the PR description).